### PR TITLE
Revert commits pushed to the main branch by accident

### PR DESCRIPTION
**Pull request by eitsupi**
See https://github.com/pola-rs/r-polars/commit/08ce8c45c2a903e375ae0d344e0065e36ee4f2b0#r117064940

### CODE REVIEW

**Comment by etiennebacher** [comment]
the problem is that `melt` and some others were already merged in `main` but I forgot to update my `main` before merging which is why I had to fix the conflict. Since the CI was fine with my 2 commits, can't we just let them there?

**Comment by eitsupi** [comment]
> the problem is that `melt` and some others were already merged in `main` but I forgot to update my `main` before merging which is why I had to fix the conflict. Since the CI was fine with my 2 commits, can't we just let them there?

I cannot determine what has been changed because at first glance I do not know what has been changed

I am fine either way, but I believe that submitting it again as a Pull Request is the more reliable way.
If we follow the changes later, we will not really know what happened.

**Comment by sorhawell** [comment]
That resolve conflicts have tricked me before also.

Ok so we remove option to override in branchrules. And I guess if any admin really really need to override some day, then admin must temporaily toggle the branch rule.

**Comment by etiennebacher** [comment]
I will redo a clean PR with these changes, it's not too big anyway

**Comment by etiennebacher** [comment]
But @eitsupi I think reverting this goes back too early, it was already 0.6.1 before my first commit.

**Comment by eitsupi** [comment]
> But @eitsupi I think reverting this goes back too early, it was already 0.6.1 before my first commit.

@etiennebacher I'm sorry but I don't understand what you are talking about. It would be very helpful if you could do this.

**Comment by sorhawell** [comment]
@etiennebacher just point to a PR which is a fair try. Then we can all review it and add any changes. If not exactly the same so be it.

**Comment by eitsupi** [comment]
Would it be better to force push to the main branch?

File: DESCRIPTION [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Package: polars
 Title: Polars ported to R
-Version: 0.6.1.9000
+Version: 0.6.0.9000
```
Comment (eitsupi) [inline DESCRIPTION:3]: @etiennebacher I see, perhaps you have a point about here?
I really dont know how to undo this other than force push.

**Comment by etiennebacher** [comment]
Thank you @eitsupi I think the last force-push solved it, this PR now goes back to the state before my last 2 commits on `main`. I think this can be merged and I will make a new PR with my 2 commits

**Comment by eitsupi** [comment]
Ok, I finally could back to 0.6.1 with this command:

```sh
git diff 08ce8c45c2a903e375ae0d344e0065e36ee4f2b0..ddf5510b47ada712ff7838cac74bfbf3f5a226dd >to_fix.diff
mv to_fix.diff /tmp/to_fix.diff
git apply /tmp/to_fix.diff
```

4b897bcd67a36f831b4707fac7a629789c72ddb8 is completely identical to 0.6.1

```sh
git diff 4b897bcd67a36f831b4707fac7a629789c72ddb8..ddf5510b47ada712ff7838cac74bfbf3f5a226dd
```

**Review by etiennebacher — APPROVED** [review]

**Comment by etiennebacher** [comment]
FYI I think a simpler solution would have been `git reset ddf5510b47ada712ff7838cac74bfbf3f5a226dd` (last "good" commit)

**Comment by eitsupi** [comment]
Thanks @etiennebacher, It would be sufficient to simply create a PR on the branch where you revert the commit that merged this.

**Comment by eitsupi** [comment]
> FYI I think a simpler solution would have been git reset ddf5510b47ada712ff7838cac74bfbf3f5a226dd (last "good" commit)

My understanding is that `git reset` only moves the tree and does not create a new commit.

**Pull request merged**